x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_page/alerts_page.tsx

```diff
@@ -135,29 +136,9 @@ const AlertsPage: React.FunctionComponent = () => {
   };
 
   const tableProps = {
+    ownerPluginId: PLUGIN_ID,
```

Comment:
nit: To avoid confusion with the case's owner let's call the prop `pluginId` or similar.

Reply:
It sounds better, will do.